packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -7,8 +7,9 @@ import {
   ExperienceParams,
   getPersonalizedRewrite,
 } from '@sitecore-jss/sitecore-jss/personalize';
-import { SiteResolver } from '@sitecore-jss/sitecore-jss/site';
+import { SiteInfo, SiteResolver } from '@sitecore-jss/sitecore-jss/site';
 import { debug, NativeDataFetcher } from '@sitecore-jss/sitecore-jss';
+import { resolvePointOfSale } from '../utils';
```

Comment:
This shouldn't work - need to import `PosResolver` from `@sitecore-jss/sitecore-jss/personalize` now, right?